src/main/webapp/course.html

```diff
@@ -246,11 +230,28 @@ <h1 class="p-4 text-center">Add Your Review</h1>
       integrity="sha384-XEerZL0cuoUbHE4nZReLT7nx9gQrQreJekYhJD9WNWhH8nEW+0c5qq7aIo2Wl30J"
       crossorigin="anonymous"
     ></script>
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.1/css/bootstrap.min.css"
+      integrity="sha384-VCmXjywReHh4PwowAiWNagnWcLhlEJLA5buUprzK8rxFgeH0kww/aWY76TfkUoSX"
+      crossorigin="anonymous"
+    />
     <script
       type="text/javascript"
       src="https://www.gstatic.com/charts/loader.js"
     ></script>
-    <script src="dummyData.js"></script>
```

Comment:
why is this being deleted?

Reply:
Ohh, I assumed this one was being used for testing because of the name. I will add it back, but could you change the name of the file by any chance? It is a bit confusingg